easybuild/easyconfigs/s/svFSI/svFSI-2022.09.26-foss-2021b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'svFSI'
+version = '2022.09.26'
+
+homepage = 'https://github.com/SimVascular/svFSI'
+description = "A multi-physics finite element solver for patient-specific blood flow simulation including fluid-structure interaction and cardiac electrophysiology Resources"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/SimVascular/%(name)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    '%(name)s-%(version)s_cmake-rpath-fix.patch',
+]
+checksums = [
+    '4f4fbf4112402c7d037f35043f8aa77b141276a59ca957614cf8de20543bd6eb',  # 2022.09.26.tar.gz
+    'be5a4889167cf242d863fa424ade7d8556c1d3d9525a809a230611a14ab095f8',  # svFSI-2022.09.26_cmake-rpath-fix.patch
+]
+
+separate_build_dir = True
+
+builddependencies = [('CMake', '3.22.1')]
```

Comment:
we should perhaps also add the optional dependencies that `svFSI` supports?
`Boost`, `HDF5`, `Hypre`, `MUMPS`, `Trilinos` should all have existing easyconfigs

Reply:
@jfgrimm Unless I'm missing something, it looks like all those deps are related to Trilinos. I didn't see an easybuild recipe for Trilinos for foss-2021b which is why I was inclined to leave it off this recipe. Let me know what you think?